libnmstate/nm/ipv6.py

```diff
@@ -214,12 +206,12 @@ def get_ip_profile(active_connection):
     return None
 
 
-def get_route_running():
-    return nm_route.get_running(_acs_and_ip_cfgs(nmclient.client()))
+def get_route_running(client):
```

Comment:
Just for consistency with the other usages, is this suppose to be `nm_client`?
(same for the other occurrences in this module)